docs/wiki/NuGet-packages.md

```diff
@@ -42,6 +42,8 @@ Below are the versions of the language available in the NuGet packages. Remember
 
 See the [history of C# language features](https://github.com/dotnet/csharplang/blob/main/Language-Version-History.md) for more details.
 
+See the [.NET SDK, MSBuild, and Visual Studio versioning](https://docs.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) docs page for details of SDK versioning. 
```

Comment:
To redirect users to page with their preferred language.

```suggestion
See the [.NET SDK, MSBuild, and Visual Studio versioning](https://docs.microsoft.com/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) docs page for details of SDK versioning. 
```

Reply:
Good idea, thanks